File: pas-rest/src/main/kotlin/model/UserRole.java
package model;

public enum UserRole {
    ADMIN, WORKER, CLIENT;

    public static UserRole fromString(String role) {
        if (role.equalsIgnoreCase("ADMIN")) {
            return ADMIN;
        } else if (role.equalsIgnoreCase("WORKER")) {
            return WORKER;
        } else {
            return CLIENT;
        }
    }
}

File: tks/ApplicationCore/src/main/kotlin/model/UserRole.kt
package model

import java.lang.Exception

enum class UserRole {
    ADMIN, WORKER, CLIENT;

    companion object{
        @JvmStatic
        fun fromString(role: String): UserRole {
            return when(role){
                "ADMIN" -> ADMIN
                "WORKER" -> WORKER
                "CLIENT" -> CLIENT
                else -> throw Exception()
            }
        }
    }
}